spec/support/helpers/general_spec_helpers.rb

```diff
@@ -50,4 +62,13 @@ def with_env(vars)
   ensure
     vars.each_pair { |k, _| ENV.delete(k) }
   end
+
+  private
+
+  def fixture(dir, name)
+    path = Rails.root.join("spec/fixtures/#{dir}/#{name}")
```

Comment:
Nice! Note the proper way to use join is `join("spec", "fixtures", dir, name)`. That way you're not dependent on the `/`. But it's not a big deal.